### PR TITLE
Feature/oda 948 eligibility criteria csv

### PR DESCRIPTION
**Pull request by dexterawoyemiGE**

### CODE REVIEW

File: ModelCatalogueGenomicsPlugin/test/integration/org/modelcatalogue/gel/GelCsvExporterSpec.groovy [hunk 1]
```diff
@@ -1,83 +0,0 @@
...
-import org.modelcatalogue.core.DataClass
-import org.modelcatalogue.core.DataModelService
-import org.modelcatalogue.core.ElementService
-import org.modelcatalogue.core.InitCatalogueService
-import org.modelcatalogue.core.util.builder.DefaultCatalogueBuilder
-
-/**
- * Created by dexterawoyemi on 21/03/2016.
- */
-class GelCsvExporterSpec extends IntegrationSpec {
```
Comment (musketyr) [inline ModelCatalogueGenomicsPlugin/test/integration/org/modelcatalogue/gel/GelCsvExporterSpec.groovy]: @dexterawoyemiGE is there any reason to delete the spec completely or have you just forgot to commit the renamed spec?

**Comment by musketyr** [comment]
there is an error in [RareDiseaseCsvExporterSpec](http://mc-travis-results.orany.cz.s3.amazonaws.com/MetadataRegistry/ModelCataloguePlugin/1730/1730.4/home/travis/reports/gel-integration-tests-reports/test-reports/html/org/modelcatalogue/gel/1_RareDiseaseCsvExporterSpec.html)

**Comment by dexterawoyemiGE** [comment]
Sorry, forgot to check GitHub messages. Yes, test was renamed, I have now modified the test to work with abstracted data from Rick's changes

**Comment by musketyr** [comment]
cool, thanks you! I don't know anything about how does the export should look like but I trust your test 😃

**Comment by dexterawoyemiGE** [comment]
ha, great thanks!

**Pull request merged**